docs/backends/armeria.md

```diff
@@ -0,0 +1,21 @@
+# Armeria backend
```

Comment:
I'd add a section to `Future-based backends` instead. There's nothing that is *that* special abour armeria that it would need a separate section